### ISSUE TITLE
OSM Wizard muss generate netedit configs

### ISSUE THREAD

**Issue by palvarezlopez** [eclipse/sumo#12843]

**Comment by namdre** [comment]
why can't the user open the sumocfg instead? (FYI: I've never felt the need for a netedit config outside of testing).

**Comment by palvarezlopez** [comment]
@namdre because there is certain options (for example, meanData files), that only exist in netedit options. Also it could be non-intuitive for users (It looks like opening a Word document using Excel)

**Comment by namdre** [comment]
I agree that there is a theoretical use case (i.e. if you repeatedly want to edit the same data file). Still, I've never encountered this. Also, I'm pretty sure that it's way beyond of what first-time users who the wizard targets find helpful. Instead I'm a bit afraid that it's one more file that adds to being overwhelmed by the osmWebWizard outputs.
